parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -60,10 +60,13 @@
         {
             "platformFaultDomainCount": 2,
             "platformUpdateDomainCount": 3,
-		"managed" : "true"
+    "managed" : "true"
         },
 
-      "type": "Microsoft.Compute/availabilitySets"
+      "type": "Microsoft.Compute/availabilitySets",
```

Comment:
Hey @ritazh @sozercan @khenidak I want to add the VMs directly to the scale set, but they're created with a loop. Is there a way to refer to instances of count in a template? Specifically, I'm trying to refer to this loop in the availability set: https://github.com/Azure/acs-engine/blob/412335166c7ab11e7e36eb7eb5da272dd6edf433/parts/k8s/kubernetesagentresourcesvmas.t#L129

Reply:
If you mean thru arm template, then you can try `copyIndex(variables('{{.Name}}Offset'))` to get the index.